setup.py

```diff
@@ -0,0 +1,302 @@
+
+    Instead of copying the files to the expected directories, a symlink is
+    created on the system aiming the current source code.
+    """
+
+    description = "Install NApps in development mode"
+
+    def run(self):
+        """Install the package in a developer mode."""
+        super().run()
+        if self.uninstall:
+            shutil.rmtree(str(ENABLED_PATH), ignore_errors=True)
+        else:
+            self._create_folder_symlinks()
+            # self._create_file_symlinks()
+            KytosInstall.enable_core_napps()
+
+    @staticmethod
+    def _create_folder_symlinks():
+        """Symlink to all Kytos NApps folders.
+
+        ./napps/kytos/napp_name will generate a link in
+        var/lib/kytos/napps/.installed/kytos/napp_name.
+        """
+        links = INSTALLED_PATH / "kytos"
+        links.mkdir(parents=True, exist_ok=True)
+        code = CURRENT_DIR
+        src = links / NAPP_NAME
+        symlink_if_different(src, code)
+
+        (ENABLED_PATH / "kytos").mkdir(parents=True, exist_ok=True)
+        dst = ENABLED_PATH / Path("kytos", NAPP_NAME)
+        symlink_if_different(dst, src)
+
+    @staticmethod
+    def _create_file_symlinks():
+        """Symlink to required files."""
+        src = ENABLED_PATH / "__init__.py"
+        dst = CURRENT_DIR / "napps" / "__init__.py"
+        symlink_if_different(src, dst)
+
+
+def symlink_if_different(path, target):
+    """Force symlink creation if it points anywhere else."""
+    # print(f"symlinking {path} to target: {target}...", end=" ")
+    if not path.exists():
+        # print(f"path doesn't exist. linking...")
+        path.symlink_to(target)
+    elif not path.samefile(target):
+        # print(f"path exists, but is different. removing and linking...")
+        # Exists but points to a different file, so let's replace it
+        path.unlink()
+        path.symlink_to(target)
+
+
+def read_version_from_json():
+    """Read the NApp version from NApp kytos.json file."""
+    file = Path('kytos.json')
+    metadata = json.loads(file.read_text(encoding="utf8"))
+    return metadata['version']
+
+
+def read_requirements(path="requirements/run.txt"):
+    """Read requirements file and return a list."""
+    with open(path, "r", encoding="utf8") as file:
+        return [
+            line.strip()
+            for line in file.readlines()
+            if not line.startswith("#")
+        ]
+
+
+setup(
+    name=f"kytos_{NAPP_NAME}",
+    version=read_version_from_json(),
+    description="This NApp implements Oplenflow multi tables",
+    url="http://github.com/kytos/{NAPP_NAME}",
+    author="Kytos Team",
+    author_email="of-ng-dev@ncc.unesp.br",
+    license="MIT",
+    install_requires=read_requirements() + ["setuptools >= 36.0.1"],
+    packages=[],
+    extras_require={
+        "dev": [
+            "pytest==7.0.0",
+            "pytest-cov==3.0.0",
+            "pip-tools",
+            "yala",
+            "tox",
```

Comment:
```suggestion
            "tox==3.28.0",
```

In the CI we're still using the last version of tox v3, and [v4 isn't fully supported yet](https://github.com/kytos-ng/kytos/issues/313), and since we reuse a global template on scrutinizer we'd like to keep v3 here too, could you please pin this version here and regenerate requirements/dev.txt?

Reply:
Added